.gitignore

```diff
@@ -169,6 +149,7 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+!crates/ruff_python_resolver/resources/test/airflow/venv
```

Comment:
Can we move this up to the project-specific ignores at the top?

Reply:
unfortunately not, the reinclusion needs to happen after the exclusion